examples/samplers/simulated_annealing_sampler.py

```diff
@@ -1,6 +1,7 @@
 """
-Optuna example that implements a custom relative sampler based on Simulated Annealing algorithm.
-Please refer to https://en.wikipedia.org/wiki/Simulated_annealing for Simulated Annealing itself.
+Optuna example that implements an user defined relative sampler based on Simulated Annealing
```

Comment:
`a user` seems to be frequently seen than `an user`.
https://www.quora.com/Is-it-an-user-or-a-user

In addition to that, `user-defined relative sampler` may be a bit easy for readers to see the dependency, in my opinion.